universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -59,6 +65,10 @@ public void putInteger(Path path, int value) {
     put(path, value);
   }
 
+  public void putLong(Path path, long value) {
+    putString(path, Long.toString(value));
+  }
```

Comment:
Issue (blocking): will this prevent numeric calculations using the JsonPath library?